content/en/docs/concepts/architecture/nodes.md

```diff
@@ -475,7 +476,7 @@ Message:        Pod was terminated in response to imminent node shutdown.
 
 ### Pod Priority based graceful node shutdown {#pod-priority-graceful-node-shutdown}
 
-{{< feature-state state="alpha" for_k8s_version="v1.23" >}}
+{{< feature-state state="beta" for_k8s_version="v1.24" >}}
```

Comment:
Everything except this line should stay the same. Your editor has tried to reformat the existing Markdown - can you find out (I mean learn) how to avoid that?

Reply:
Basically I just need to turn off the default formatting in my editor.
I will keep that in mind in future and 
Will it in this pr too

Reply:
I fixed it now!!